mcs/class/System.Core/System.IO.Pipes/AnonymousPipeServerStream.Windows.cs

```diff
@@ -0,0 +1,73 @@
+using Microsoft.Win32.SafeHandles;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace System.IO.Pipes
+{
+	public sealed partial class AnonymousPipeServerStream
+	{
+		public AnonymousPipeServerStream (PipeDirection direction, HandleInheritability inheritability, int bufferSize, PipeSecurity pipeSecurity)
```

Comment:
I am not sure about the practices on Moni repo but this file should be exactly like the one in CoreFx except for this constructor, correct? If yes, perhaps it is better to have the constructors that don't exist in CoreFx on separate files so whenever new changes are picked up from CoreFx these constructors are not involved in the diff. Same for the other locations.

Reply:
These files don't mirror CoreFX (except for the Create methods that I am about to remove once/if https://github.com/dotnet/corefx/pull/31118 is merged). The CoreFX files are linked directly by path (../../../external/corefx/src/[something]). The file names are TBD, not sure about the conventions.

Reply:
Specifically, the CoreFX files are referenced directly from various .sources files (and .csproj is generated from that). The .cs files that are left in the Mono repository fill in the missing pieces, in this case the constructors with the PipeStream parameter. Ideally these files should be as small as possible, which is not the case here just yet. It will probably take couple more iterations to get there (PRs on CoreFX upstream, PRs/cherry-picks to Mono's CoreFX forked repository and this PR).